[2024-12-30T22:04:17.458+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-30T22:04:17.474+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_kafka_producer.fetch_and_publish_weather manual__2024-12-30T22:04:16.591247+00:00 [queued]>
[2024-12-30T22:04:17.481+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_kafka_producer.fetch_and_publish_weather manual__2024-12-30T22:04:16.591247+00:00 [queued]>
[2024-12-30T22:04:17.481+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-30T22:04:17.492+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_and_publish_weather> on 2024-12-30 22:04:16.591247+00:00
[2024-12-30T22:04:17.499+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=525) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-30T22:04:17.501+0000] {standard_task_runner.py:72} INFO - Started process 527 to run task
[2024-12-30T22:04:17.501+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_kafka_producer', 'fetch_and_publish_weather', 'manual__2024-12-30T22:04:16.591247+00:00', '--job-id', '886', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmpmsbogw0e']
[2024-12-30T22:04:17.502+0000] {standard_task_runner.py:105} INFO - Job 886: Subtask fetch_and_publish_weather
[2024-12-30T22:04:17.542+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_kafka_producer.fetch_and_publish_weather manual__2024-12-30T22:04:16.591247+00:00 [running]> on host 865c783184f2
[2024-12-30T22:04:17.611+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_kafka_producer' AIRFLOW_CTX_TASK_ID='fetch_and_publish_weather' AIRFLOW_CTX_EXECUTION_DATE='2024-12-30T22:04:16.591247+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-30T22:04:16.591247+00:00'
[2024-12-30T22:04:17.612+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-30T22:04:17.820+0000] {logging_mixin.py:190} INFO - Weather data for city ID 2538474 published to Kafka successfully.
[2024-12-30T22:04:17.821+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-30T22:04:17.830+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-30T22:04:17.830+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_kafka_producer, task_id=fetch_and_publish_weather, run_id=manual__2024-12-30T22:04:16.591247+00:00, execution_date=20241230T220416, start_date=20241230T220417, end_date=20241230T220417
[2024-12-30T22:04:17.876+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-30T22:04:17.891+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-30T22:04:17.893+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
